sans/CreateConvertPlotExample.ipynb

```diff
@@ -0,0 +1,129 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def _repr_html_(input_scope=None):\n",
+    "    import inspect\n",
+    "    # Is there a better way to get the scope? The `7` is hard-coded for the\n",
+    "    # current IPython stack when calling _repr_html_ so this is bound to break.\n",
+    "    scope = input_scope if input_scope else inspect.stack()[7][0].f_globals\n",
+    "    from IPython import get_ipython\n",
+    "    ipython = get_ipython()\n",
+    "    out = ''\n",
+    "    for category in ['Variable', 'DataArray', 'Dataset']:\n",
+    "        names = ipython.magic(f\"who_ls {category}\")\n",
+    "        out += f\"<details open=\\\"open\\\"><summary>{category}s:\"\\\n",
+    "               f\"({len(names)})</summary>\"\n",
+    "        for name in names:\n",
+    "            html = make_html(eval(name, scope))\n",
+    "            out += f\"<details style=\\\"padding-left:2em\\\"><summary>\"\\\n",
+    "                   f\"{name}</summary>{html}</details>\"\n",
+    "        out += \"</details>\"\n",
+    "    from IPython.core.display import display, HTML\n",
+    "    display(HTML(out))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from convert_helpers import ConvertWidget, DataCreationWidget, PlotWidget\n",
+    "import IPython.display as disp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data creation\n",
+    "Simple widget to create some pesudo real histogram data with sample, background and positional data. Currently only supports creating time of flight data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_creation = DataCreationWidget(globals())\n",
+    "disp.display(data_creation)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data conversion"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "data_conversion = ConvertWidget(globals())\n",
+    "disp.display(data_conversion)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data plotting\n",
+    "Plots the data aquired by evaluating the expression entered."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "plot_widget = PlotWidget(globals())\n",
+    "data_creation.subscribe(plot_widget)\n",
+    "data_conversion.subscribe(plot_widget)\n",
+    "disp.display(plot_widget)"
```

Comment:
Just tried the notebook but got this error
![Screenshot at 2020-10-06 09-19-23](https://user-images.githubusercontent.com/39047984/95170674-223f9780-07b5-11eb-8483-631b6362cdd0.png)

Reply:
but maybe this is reated to Simon's other PR?

Reply:
Sorry just seen your comment above...